### PR TITLE
added Ag list for IP dashboard

### PR DESCRIPTION
**Pull request by yogesh-pawar-2023**

### CODE REVIEW

File: src/src/beneficiaries/beneficiaries.controller.ts [hunk 1]
```diff
@@ -48,6 +48,17 @@ export class BeneficiariesController {
 	) {
 		return this.beneficiariesService.findAll(request, req, response);
 	}
+
+	@Post('/ip')
```
Comment (manojLondhe) [inline src/src/beneficiaries/beneficiaries.controller.ts:52]: rename
/ip
to
/admin/list

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 2]
```diff
@@ -101,6 +101,119 @@ export class BeneficiariesService {
 		});
 	}
 
+	public async findAllAgForIp(body: any, req: any, resp: any) {
```
Comment (manojLondhe) [inline src/src/beneficiaries/beneficiaries.service.ts:104]: rename

findAllAgForIp() => getList()

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: src/src/beneficiaries/beneficiaries.service.ts (hunk 2 above)
Comment (yogesh-pawar-2023) [reply src/src/beneficiaries/beneficiaries.service.ts:104]: updated as per comment

File: src/src/beneficiaries/beneficiaries.controller.ts (hunk 1 above)
Comment (yogesh-pawar-2023) [reply src/src/beneficiaries/beneficiaries.controller.ts:52]: updated to /admin/list

**Pull request merged**